src/common_rl/to_env.jl

```diff
@@ -0,0 +1,86 @@
+const RL = CommonRLInterface
+
+abstract type AbstractPOMDPsCommonRLEnv <: RL.AbstractMarkovEnv end
+
+RL.actions(env::AbstractPOMDPsCommonRLEnv) = actions(env.m)
+RL.terminated(env::AbstractPOMDPsCommonRLEnv) = isterminal(env.m, env.s)
+
+mutable struct MDPCommonRLEnv{RLO, M<:MDP, S} <: AbstractPOMDPsCommonRLEnv
+    m::M
+    s::S
+end
+
+MDPCommonRLEnv{RLO}(m, s) where {RLO} = MDPCommonRLEnv{RLO, typeof(m), typeof(s)}(m, s)
+
+function RL.reset!(env::MDPCommonRLEnv)
+    env.s = rand(initialstate(env.m))
```

Comment:
We can't pass RNG here?

Reply:
Hmmm... yeah, eventually we should add some rng considerations to CommonRLInterface